pkgs/applications/misc/shavee/default.nix

```diff
@@ -0,0 +1,45 @@
+{ lib, pkgs, rustPlatform, fetchFromGitHub, pkg-config, libiconv, openssl, linux-pam }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "shavee";
+  version = "v0.5.1";
+
+  src = fetchFromGitHub {
+    owner = "ashuio";
+    repo = "shavee";
+    rev = "${version}";
+    sha256 = "sha256-41wJ3QBZdmCl7v/6JetXhzH2zF7tsKYMKZY1cKhByX8=";
+  };
+
+  cargoSha256 = "sha256-j0wRfRG6TU9LBhhCWf+cBxNICO1s9pyOQn0tm2Lrr0Y=";
+
+  nativeBuildInputs = [
+    pkg-config
+  ];
+
+  buildInputs = [
+      libiconv
+      linux-pam
+      openssl
+      openssl.dev
+
+  ];
+
+  buildPhase = ''
+    cargo build --release
+  '';
+
+  installPhase = ''
+                  mkdir -p $out/bin
+                  mkdir -p $out/lib
+                  cp target/release/shavee $out/bin/
+                  cp target/release/libshavee_pam.so $out/lib/
+                '';
+
+  meta = with lib; {
+    description = "A Program to automatically decrypt and mount ZFS datasets using Yubikey HMAC as 2FA or any File on USB/SFTP/HTTPS.";
+    homepage = "https://github.com/ashuio/shavee";
+    license = licenses.mit;
+    maintainers = with maintainers; [jasonodoom];
```

Comment:
there is no `jasonodoom`-attribute in `/maintainers/maintainer-list.nix`.

Reply:
It's probably better to add yourself to the maintainer list instead of adding a package without maintainer.